access-token-management/src/AccessTokenManagement/LogMessages.cs

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Duende Software. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Duende.AccessTokenManagement;
+public static partial class LogMessages
+{
+    internal class Parameters
+    {
+        public const string Scheme = "{scheme}";
+        public const string Error = "{error}";
+        public const string Url = "{url}";
+        public const string ClientName = "{clientName}";
+        public const string Expiration = "{expiration}";
+        public const string Token = "{token}";
+        public const string Endpoint = "{endpoint}";
+        public const string User = "{user}";
+        public const string Resource = "{resource}";
+        public const string Method = "{method}";
+        public const string Address = "{address}";
+    }
+
+    [LoggerMessage(
+        Level = LogLevel.Information,
+        Message = $"Cannot authenticate scheme: {Parameters.Scheme}")]
+    public static partial void InformationCannotAuthenticateScheme(
+        this ILogger logger, string scheme);
+
+    [LoggerMessage(
+        Level = LogLevel.Information,
+        Message = $"Authentication result properties are null for scheme: {Parameters.Scheme}")]
+    public static partial void InformationAuthenticationResultPropertiesAreNull(
+        this ILogger logger, string scheme);
+
+    [LoggerMessage(
+        Level = LogLevel.Information,
+        Message = "No tokens found in cookie properties. SaveTokens must be enabled for automatic token refresh.")]
+    public static partial void InformationNoTokensFoundInCookieProperties(this ILogger logger);
+
+    [LoggerMessage(
+        Level = LogLevel.Information,
+        Message = $"Error revoking refresh token. Error = {Parameters.Error}.")]
+    public static partial void InformationFailedToRefreshToken(this ILogger logger, string? error);
+
+    [LoggerMessage(
+        Level = LogLevel.Debug,
+        Message = $"Sending DPoP proof token in request to endpoint: {Parameters.Url}")]
+    public static partial void DebugSendingDPoPProofToken(this ILogger logger, string? url);
+
+    [LoggerMessage(
+        Level = LogLevel.Debug,
+        Message = $"No DPoP proof token in request to endpoint: {Parameters.Url}")]
+    public static partial void DebugNoDPoPProofToken(this ILogger logger, string? url);
+
+    [LoggerMessage(
+        EventId = 3,
```

Comment:
Why does this LoggerMessage have an EventId?

Reply:
removed it.